kinto/plugins/kinto_admin/tests/__init__.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import webtest
+from kinto.core import utils as core_utils
+
+
+def get_user_headers(user):
+    credentials = "%s:secret" % user
+    authorization = 'Basic {0}'.format(core_utils.encode64(credentials))
+    return {
+        'Authorization': authorization
+    }
+
+
+class BaseWebTest(object):
+    config = '../../../../config/kinto.ini'
```

Comment:
@Natim Can I just move the test to the tests/plugins folder where all the other plugin tests are are configure them in the second way?